lldb-eval/eval_test.cc

```diff
@@ -3405,3 +3405,43 @@ TEST_F(EvalTest, TestRegistersNoDollar) {
     EXPECT_THAT(Eval(reg_name), IsEqual(reg.GetValue()));
   }
 }
+
+TEST_F(EvalTest, TestCharParsing) {
+  EXPECT_THAT(Eval("1 + 'A'"), IsEqual("66"));
+  EXPECT_THAT(Eval("'B' - 'A'"), IsEqual("1"));
+  EXPECT_THAT(Eval("'A' == 'B'"), IsEqual("false"));
+  EXPECT_THAT(Eval("'A' == 'A'"), IsEqual("true"));
+
+  /*Following test fails detecting different results from
+   *as u8 prefix is not handled in lldb
+   *EXPECT_THAT(Eval("1 + u8'A'"), IsEqual("66"));
+   *EXPECT_THAT(Eval("u8'B' - u8'A'"), IsEqual("1"));
+   *EXPECT_THAT(Eval("u8'A' == u8'B'"), IsEqual("false"));
+   *EXPECT_THAT(Eval("u8'A' == u8'A'"), IsEqual("true"));
+   */
```

Comment:
Use single line comments -- `//`, multiline are generally more difficult to edit.